docker/readme.md

```diff
@@ -0,0 +1 @@
+A Docker application in which to build and run MAG.
```

Comment:
Can you use the same line (and address the same comment) I made here: https://github.com/geodynamics/conman/pull/10#discussion_r1559889079